backend/penndata/views.py

```diff
@@ -28,12 +31,18 @@
 )
 
 
+ValidatedData: TypeAlias = dict[str, Any]
+CalendarEventList: TypeAlias = QuerySet[CalendarEvent, Manager[CalendarEvent]]
+EventList: TypeAlias = QuerySet[Event, Manager[Event]]
+HomePageOrderList: TypeAlias = QuerySet[HomePageOrder, Manager[HomePageOrder]]
```

Comment:
Same with these. I know they are for the sake of clarity, but I feel like its a bit overkill so just adding clutter here. And also in Django world, seeing that it is a queryset is actually probably more helpful.